grblas/ops.py

```diff
@@ -191,10 +191,9 @@ class BinaryOp(OpBase):
     all_known_instances = set()
 
     @classmethod
-    def register_new(cls, name, func):
+    def _build_udf(cls, func, name=None):
```

Comment:
Hmm, probably no harm exposing this to the user, so no need to hide behind an underscore.